camera_handler.py

```diff
@@ -0,0 +1,61 @@
+import cv2
+import sys
+
+
+class CameraHandler:
+
+    def __init__(self, camera):
+        self.__width = None
+        self.__height = None
+        self.__frame_rate = None
+        self.camera = cv2.VideoCapture(camera)
+        if not self.camera.isOpened():
+            print('Error opening camera.')
+            sys.exit(1)
+
+    # This function returns the camera output width (int)
+    def get_camera_width(self):
+        return self.__width
+
```

Comment:
These getters could use opencv directly like here: https://stackoverflow.com/questions/39953263/get-video-dimension-in-python-opencv it's also possible to set the width and height this way.

Reply:
Comment or suggested change?

Reply:
If there's no specific reason to keep it like this, consider it a suggested change.

Reply:
Fixed!